Resources/public/css/theme/views/studiopresentation.css

```diff
@@ -4,7 +4,10 @@
  */
 
 .ez-view-studiopresentationview {
-    /* for test purpose */
-    padding: 1em;
+    padding: 0px;
```

Comment:
not sure you really need that but if it's the case, this should go in the layout stylesheet not in a theme one.